services/utils/__init__.py

```diff
@@ -174,6 +177,8 @@ def __init__(self,
         self.pool_min = int(os.environ.get(prefix + "POOL_MIN", self.pool_min))
         self.pool_max = int(os.environ.get(prefix + "POOL_MAX", self.pool_max))
 
+        self.timeout = int(os.environ.get(prefix + "TIMEOUT", self.timeout))
+
```

Comment:
should this default to `timeout` instead of `self.timeout` or does it work as intended?